core/modules/config/tests/config.test

```diff
@@ -548,6 +562,16 @@ class ConfigurationUITest extends BackdropWebTestCase {
 class ConfigurationSyncTest extends BackdropWebTestCase {
   protected $profile = 'testing';
 
+  /**
+   * @var object
+   */
+  protected $content_type;
```

Comment:
This may not be needed at all? `$content_type` is assigned in the `setUp()` method, but I don't see why. The test could create the content type simply by invoking 

```php
$this->backdropCreateContentType(array(
    'type' => 'post',
    'name' => 'Post',
));
```
without assigning its return value to anything. 

I can see how changing this (removing unused or unnecessary dynamic properties) may be beyond the scope here. But just commenting in case you feel this is worth considering.

Reply:
Yep, that's a clearly redundant one. :+1: